contracts/crowdsale/distribution/PostDeliveryRefundableCrowdsale.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.4.24;
+
+import "./PostDeliveryCrowdsale.sol";
+import "./RefundableCrowdsale.sol";
+
+
+/**
+ * @title PostDeliveryRefundableCrowdsale
+ * @dev Extension of RefundableCrowdsale contract that only delivers the tokens
+ * once the crowdsale has closed and the goal met, preventing refunds to be issued
+ * to token holders.
+ */
+contract PostDeliveryRefundableCrowdsale is PostDeliveryCrowdsale, RefundableCrowdsale {
```

Comment:
I think I'd go with `RefundablePostDelivery` instead of the other way around. `Refundable` is the important thing.